src/closedWindows/closedWindowList/closedWindowList-controller.js

```diff
@@ -39,6 +45,10 @@
             this.$scope.$on('$destroy', () => {
                 window.windowService.removeClosedWindowListener(listener);
             });
+
+            this.$scope.$on('windowClosed', () => {
+                this.overriddenIcon = this.$scope.icon + '_active';
+            });
```

Comment:
While it's not tidy and needs a redesign, those listeners are listening for a `closedWindowChange` event, the new one is waiting for `windowClosed`